tutorials/install.md

```diff
@@ -21,7 +21,12 @@ sudo sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `ls
 
 Setup keys:
 ```
-wget http://packages.osrfoundation.org/gazebo.key -O - | sudo apt-key add -
+wget -qO - http://packages.osrfoundation.org/gazebo.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/gazebo-archive-keyring.gpg > /dev/null
```

Comment:
Other instructions, ([example](https://gazebosim.org/docs/latest/install_ubuntu/)) use the following. I'd suggest we do that here as well for consistency.

```
sudo curl https://packages.osrfoundation.org/gazebo.gpg --output /usr/share/keyrings/pkgs-osrf-archive-keyring.gpg
echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/pkgs-osrf-archive-keyring.gpg] http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/gazebo-stable.list > /dev/null
```